screens/HomeAuth.js

```diff
@@ -109,6 +98,32 @@ class HomeAuth extends Component {
     });
   }
 
+  async initGoogleLogin() {
+    return Google.logInAsync({
+      androidClientId: CONFIG.ANDROID_CLIENT_ID,
+      // iosClientId: YOUR_CLIENT_ID_HERE,  <-- if you use iOS
```

Comment:
Can we have both of these set? Probably easiest so no-one has to change it here, just the config file

Reply:
fixed